.github/workflows/update-ims.js

```diff
@@ -0,0 +1,114 @@
+const https = require('https');
+const { execSync } = require('child_process');
+const fs = require('fs');
+
+const fetchImsScript = () =>
+  new Promise((resolve, reject) => {
+    https
+      .get('https://auth.services.adobe.com/imslib/imslib.min.js', (res) => {
+        if (res.statusCode < 200 || res.statusCode >= 300) {
+          return reject(new Error('statusCode=' + res.statusCode));
+        }
+
+        let data = '';
+
+        res.on('data', (chunk) => {
+          data += chunk;
+        });
+
+        res.on('end', () => {
+          resolve({
+            data,
+            headers: res.headers,
+          });
+        });
+      })
+      .on('error', (err) => {
+        reject(err);
```

Comment:
I'm guessing this is an actual `Error`, not a string, right?

Reply:
Yup it's an actual `Error`